airo-camera-toolkit/airo_camera_toolkit/point_clouds/conversions.py

```diff
@@ -0,0 +1,52 @@
+from typing import Any
+
+import open3d as o3d
+import open3d.core as o3c
+from airo_typing import PointCloud
+
+
+def point_cloud_to_open3d(point_cloud: PointCloud) -> Any:  # TODO: change Any back to o3d.t.geometry.PointCloud
```

Comment:
why is the o3d type not used here?

Reply:
I don't remember the exact reason, but I guess mypy didn't like it.